systemd/timesyncd/config.sls

```diff
@@ -0,0 +1,57 @@
+include:
+  - systemd.reload
+
+{%- from "systemd/map.jinja" import systemd with context %}
+
+{%- set timesyncd = systemd.get('timesyncd', {}) %}
+{%- set timezone = timesyncd.get('timezone', 'UTC') %}
+{%- set config = timesyncd.get('config', {}) %}
+
+{%- set virtual = salt['grains.get']('virtual') | default('physical', True) %}
+{%- set virtual_subtype = salt['grains.get']('virtual_subtype') | default('', True) %}
+
+timesyncd:
+  {%- if timesyncd.pkg %}
+  pkg.installed:
+    - name: {{ timesyncd.pkg }}
+  {%- endif %}
+  ini.options_present:
+    - name: /etc/systemd/timesyncd.conf
+    - separator: '='
+    - strict: True
+    - sections:
+        Time:
+          NTP: {{ config.NTP | yaml }}
+          FallbackNTP: {{ config.FallbackNTP | yaml }}
+          RootDistanceMaxSec: {{ config.RootDistanceMaxSec | yaml }}
+          PollIntervalMinSec: {{ config.PollIntervalMinSec | yaml }}
+          PollIntervalMaxSec: {{ config.PollIntervalMaxSec | yaml }}
```

Comment:
Similar to #28, do you need the following?

```sls
    - listen_in:
      - service: timesyncd
```

Reply:
yes you are correct.